yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1475,6 +1479,8 @@ func considerQueryForIssueDetection(collectedSchemaList []string) bool {
 }
 
 const (
+	RANGE_SHARDED_INDEXES_RECOMMENDATION = `If indexes are created on columns commonly used in range-based queries (e.g. timestamp columns), it is recommended to explicitly configure these indexes with range sharding. This ensures efficient data access for range queries.
+By default, YugabyteDB uses hash sharding for indexes, which distributes data randomly and is not ideal for range-based predicates potentially degrading query performance. Note that range sharding is enabled by default only in PostgreSQL compatibility mode in YugabyteDB.`
```

Comment:
can we add the link to docs for pg compatibility mode? Or perhaps we can include this in our docs

Reply:
I was thinking about it but was avoiding links because of json as well but I think it makes sense to have a link here in the notes

Reply:
Yeah, it's okay either way. we can have it in the docs too.